datalad/distribution/tests/test_uninstall.py

```diff
@@ -402,8 +412,21 @@ def test_kill(path):
     res = ds.remove(on_failure='ignore')
     assert_result_count(
         res, 1,
-        status='error', path=testfile,
-        message='configured minimum number of copies not found')
+        status='error', path=testfile)
+    # Following two assertions on message are relying on the actual error.
+    # We have a second result with status 'impossible' for the ds, that we need
+    # to filter out for those assertions:
+    err_result = [r for r in res if r['status'] == 'error'][0]
+    assert_result_values_cond(
+            [err_result], 'message',
+            lambda x: "(Use --force to override this check, "
+                      "or adjust numcopies.)" in x
+    )
```

Comment:
the same here (as below):
`assert_result_require_numcopies_or_force(err_result, 'message')`
may be even making `message` to be the default field